frontend/shared-components/data-table/data-table.tsx

```diff
@@ -0,0 +1,178 @@
+  const [rowsShown, setRowsShown] = React.useState(7)
+
+  // TODO: When this gets changed from mocking to fetching the data from an api call, append
+  // the full and save values dynamically
+  const data = React.useMemo(
+    () => [
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+      {
+        dates: "2003/01/01",
+        incidentType: "Use of force",
+        officersInvolved: ["John Smith"],
+        subject: "unknown",
+        source: "News Article",
+        full: faGreaterThan,
+        save: faPlusCircle,
+      },
+    ],
+    []
+  )
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: "Date(s)",
+        accessor: "dates" as const,
+      },
+      {
+        Header: "Incident Type",
+        accessor: "incidentType" as const,
+      },
+      {
+        Header: "Officer(s) Involved",
+        accessor: "officersInvolved" as const,
+      },
+      {
+        Header: "Subject",
+        accessor: "subject" as const,
+      },
+      {
+        Header: "Source",
+        accessor: "source" as const,
+      },
+      {
+        Header: "Full Record",
+        accessor: "full" as const,
+      },
+      {
+        Header: "Save Record",
+        accessor: "save" as const,
+      },
+    ],
+    []
+  )
+  const tableInstance = useTable({ columns, data })
+
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = tableInstance
+
+  return (
+    <div className={dTableWrapper}>
```

Comment:
@srsexton94: Should I add a <caption> to this table, perhaps on that's visually hidden?